test/json-schema.test.js

```diff
@@ -106,3 +106,58 @@ test('support file in json schema', async t => {
     type: 'file'
   }])
 })
+
+test('support response description', async t => {
+  const opts8 = {
+    schema: {
+      response: {
+        200: {
```

Comment:
Could you add a test with `'2xx'`?
That pattern is used a lot and there is a conversion to manage it

Reply:
Hi @Eomm 

When tried to use '2xx' in the response, the swagger validator fails with:

```
Swagger schema validation failed.    
Additional properties not allowed: 2xx at #/paths///get/responses   JSON_OBJECT_VALIDATION_FAILED
```

Any idea?

Reply:
I checked the schema validator of Swagger (v2) and I see "^([0-9]{3})$|^(default)$".  This regex doesn't seem to support the "2xx".
I miss something?

Reply:
```{0-9}XX``` is the syntax from OpenAPI V3 which is not allowed from Swagger Schema which use OpenAPI V2.

Reply:
@Eomm we need to make a clear distinction between v2 and v3. :(

I think it is safe to use 200 for this test.

Reply:
@Eomm this plugin supports OpenAPI v3?

Reply:
We use `2xx`  in Fastify for specifying those patterns. This module can receive `2xx` inside the schema defined for the routes.

Reply:
I found that a [bug](https://github.com/fastify/fastify-swagger/issues/250) report is exist for ```2XX``` response

Reply:
Thanks, @climba03003, I totally forgot that issue and I thought it was supported actually.. my bad

So I think the `2xx` should be fixed by another PR and this one is ok for the task it wants to accomplish 👍